### ISSUE TITLE
Backglass moves, when using a normal (non-VR Room) VPX table and a B2s backglass file

### ISSUE THREAD

**Issue by RGYsr** [vpinball/vpinball#214]
**Describe the bug**
Backglass movement with Analog nudging or headset movement, only when using a normal VPX table and a B2s backglass file
The movement is limited to approx 2 inches in any direction.

**Versions**
VPinballX_GL-10.8.0-607-eabc079-Release-gl-win-x86
VPinMAME-sc-3.6-239-0a38627-win-x86 (same result with versions 3.4 and 3.5)
DMDext-v1.10.2-x86 ((same result with version 1.8.2)
Flexdmd V1.8
B2SBackglassServer-1.4.1-589cc4e-Release-win-x86  (same result with versions1.3.1.1) 

 - Windows 10 version 22H2 X64
 - Nvidia RTX 2080 ti
 - Intel I7 4790 4Ghz
 - 16gb RAM
 - Valve Index, & Steam

**Comment by gear323** [comment]
I don't normally play many NON-VR Room tables on VR but I decided to give it a try to see if I could reproduce this issue.  

I downloaded a recent table from JP (linked below) and was indeed able to reproduce the issue. 

 https://www.vpforums.org/index.php?app=downloads&showfile=16943&fbclid=IwAR2n2Fr16JKeAzSqp6TwExds5RCmphdpBUyM1xXCbgBuSx88jbAwc6kncfM

An interesting finding is that I have "DisableB2s" checked off but the backglass does show up in VR anyway.   If I delete the B2S backglass file then I get no backglass at all.

**Comment by RGYsr** [comment]
Gear 323, Are you using steam for your headset, or some other software?

**Comment by gear323** [comment]
Sill using the same headset you tried a couple years ago now  :)  

Valve Index and Steam VR

**Comment by RGYsr** [comment]
I just tried version VPinballX_GL-10.8.0-712-292e2b7-Release-win-x86, and the movement of the backglass has been corrected. Now when nudging the whole cabinet moves slightly in sync with the nudging as a real pinball machine would. This happens on VR Room and non VR Room tables. Thanks for the fix.
There are still non VR Room tables that are newer versions of VRR tables or tables not yet in a VRR format that I want to play in VR. I only use the VPX_gl version on my cab I can't play in flat screen anymore! JP's tables  or originals for instance.
